roboteam_ai/src/skills/InterceptBall.cpp

```diff
@@ -68,7 +79,7 @@ InterceptBall::Status InterceptBall::onUpdate() {
 
 void InterceptBall::checkProgression() {
     if (keeper) {
-        if (ballInGoal()) {
+        if (ballInGoal()||missBall(ballStartPos, ballEndPos,ballStartVel)) {
```

Comment:
you would make me very happy if  you would refactor `missBall` to `missedBall` or `didMissBall` ;-)